app_modules/document_downloader.py

```diff
@@ -0,0 +1,53 @@
+from flask_restful import abort
+import requests
+
+def parse_biorxiv_document(user_doi):
+    """
+    This function is designed to render the paper-journal 
+    network for the user given a biorxiv doi
+    
+    Args:
+        user_doi - a biorxiv doi that grabs the most current version of a preprint
+    """
+    r = requests.get(
```

Comment:
You may want to use a longer variable name (such as `response`). `r` is a little too short.

Same thing on line 33.